test/unit/TokenTest.php

```diff
@@ -221,6 +221,21 @@ public function getClaimShouldRaiseExceptionWhenClaimIsNotConfigured()
         $token->getClaim('testing');
     }
 
+    /**
+     * @test
+     *
+     * @uses Lcobucci\JWT\Token::__construct
+     * @uses Lcobucci\JWT\Token::hasClaim
+     * @uses Lcobucci\JWT\Claim\Basic
+     *
+     * @covers Lcobucci\JWT\Token::getClaim
+     */
+    public function getClaimShouldReturnNullValueWhenDefaultParameterIsPassed()
+    {
+        $token = new Token();
+        $this->assertNull($token->getClaim('testing', null));
```

Comment:
Just a small nitpick, this method is static. Would you please update the call to `self::assertNull()`?

Reply:
Sure, done :+1:

Reply:
Awesome, thanks!